src/scss/components/form/_index.scss

```diff
@@ -151,5 +151,40 @@ $form-asterisk-offset: $form-asterisk-size + $form-asterisk-gap;
   }
 }
 
+hx-checkbox-control {
+  > input[type="text"] {
+    height: 32px;
+    font-size: 16px;
+    line-height: 1.5;
+    color: $gray-900;
+    background-color: $gray-0;
+    border: 1px solid $gray-500;
+    grid-area: control;
+    padding: 0 0.75rem;
+    font-family: inherit;
+  }
+
+  > input[type="text"]:focus {
+    border: 1px solid $cyan-700;
+    box-shadow: $focus-glow;
+    outline: 0;
+  }
+}
```

Comment:
Is this needed?

Reply:
It is input filed focusing style, It is not required I will be remove it.